client/my-sites/people/invite-people/index.jsx

```diff
@@ -73,14 +73,7 @@ class InvitePeople extends Component {
 	};
 
 	getInitialState = () => {
-		const { isAtomic, isWPForTeamsSite } = this.props;
-
-		let defaultRole = 'follower';
-		if ( isWPForTeamsSite ) {
-			defaultRole = 'editor';
-		} else if ( isAtomic ) {
-			defaultRole = 'subscriber';
-		}
+		const defaultRole = 'editor';
```

Comment:
Not sure I follow the changes here, "subscriber" is also an internal WP role unrelated to the concept of email subscribers.

Reply:
There is a request to remove 'Follower' and 'Subscriber' roles from the list of roles while you are adding new users. Here is the ticket: https://github.com/Automattic/wp-calypso/issues/65776

Because of that, we need to provide a default role, which is in this case 'editor'. 

The screen is:
<img width="750" alt="180416652-90bc172f-a83d-4142-b542-65d0e26576dc" src="https://user-images.githubusercontent.com/1241413/186437384-dfe20991-0cb1-4edb-a6f6-f1a2275906f4.png">